src/config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  PORT: process.env.PORT || 8000,
+  NODE_ENV: process.env.NODE_ENV || 'development',
+  DB_URL: process.env.DATABASE_URL || 'postgresql://thingful@localhost/bug-trapper',
+  TEST_DB_URL: process.env.TEST_DATABASE_URL ||'postgresql://thingful@localhost/bug-trapper-test',
```

Comment:
Let's change these names for the current project

Reply:
Why do we want to change these name?

Reply:
My branch actually has a config file with a different database name